Chap_Introduction.tex

```diff
@@ -143,71 +74,120 @@ \section{PMIx Architecture Overview}
 \endgroup
 
 
-Thus, as the diagram in Fig.~\ref{fig:roles} shows, the application is built against a \ac{PMIx} client library that contains the client-side \acp{API},
-attribute definitions, and communication support for interacting with the local \ac{PMIx} server. Intra-process cross-library interactions
-are supported at the client level to avoid unnecessary burdens on the server. Orchestration requests are sent to the
-local \ac{PMIx} server, which subsequently passes them to the host \ac{SMS} (here represented by an \ac{RM} daemon) using the \ac{PMIx} server callback functions the host \ac{SMS} registered during PMIx\_server\_init. The host \ac{SMS} can indicate its lack of support for any operation by simply providing a \textit{NULL} for the associated callback function, or can create a function entry that returns \textit{not supported} when called.
-
-The conceptual model places the burden of fulfilling the request on the host \ac{SMS}. This includes performing any
-inter-node communications, or interacting with other \ac{SMS} elements. Thus, a client request for a network traffic report
-does not go directly from the client to the \ac{FM}, but instead is relayed to the \ac{PMIx} server, and then passed to the host \ac{SMS}
-for execution. This architecture reflects the second principle underlying the standard --- namely, that connectivity is to be minimized by channeling all application interactions with the \ac{SMS} through the local \ac{PMIx} server.
+Fig.~\ref{fig:roles} shows a typical \ac{PMIx} implementation in which the application is 
+built against a \ac{PMIx} client library that contains the client-side \acp{API},
+attribute definitions, and communication support for interacting with the local \ac{PMIx} server. 
+\ac{PMIx} clients are processes which are started through the \ac{PMIx} infrastructure, 
+either by the PMIx implementation directly or through a \ac{SMS} component, and have registered 
+as clients.  A \ac{PMIx} client 
+is created in such a way that the \ac{PMIx} client library
+will be have sufficient information available to
+authenticate with the \ac{PMIx} server.  
+The \ac{PMIx} server will have sufficient knowledge about the  
+process which it created, either directly or through other \ac{SMS}, to authenticate the 
+process and provide information the process requests such as its identity and the 
+identity of its peers.  
+
+As clients invoke \ac{PMIx} \acp{API}, it is possible that some client requests can 
+be handled at the client level.  Other requests might require communication with the 
+local \ac{PMIx} server, which subsequently might request services from the host \ac{SMS} 
+(represented here by a \ac{RM} daemon).  
+The interaction between the \ac{PMIx} server and \ac{SMS} are 
+achieved using callback functions registered during server initialization.  
```

Comment:
An SMS may choose to completely embed the PMIx server; thus not needing the PMIx server API. 

Maybe different wording would work better;
```suggestion
To facilitate the interaction between the \ac{PMIx} server and the \ac{SMS}, the \ac{PMIx} server API provides callback functions to be registered during server initialization.
```